Bio/SeqUtils/NucUtils.py

```diff
@@ -0,0 +1,191 @@
+# Copyright 2018 Aleksandra Jarmolinska.  All rights reserved.
+# This file is part of the Biopython distribution and governed by your
+# choice of the "Biopython License Agreement" or the "BSD 3-Clause License".
+# Please see the LICENSE file that should have been included as part of this
+# package.
+"""Functions for dealing with GC/AT content."""
+
+
+from __future__ import print_function
+
+__GC_CONTENT_VALUE = {
+    "G": 1.,
+    "C": 1.,
+    "A": 0.,
+    "T": 0.,
+    "S": 1.,
+    "W": 0.,
+    "N": 0.5
+}
+__GC_CONTENT_VALUE_AMBG = {
+    "M": 0.5,
+    "K": 0.5,
+    "R": 0.5,
+    "Y": 0.5,
+
+    "B": 0.66,
+    "D": 0.33,
+    "H": 0.33,
+    "V": 0.66,
+}
```

Comment:
Here I would be much happier if the magic values here computed on the fly when first used from the ambiguity tables in ``Bio/Data/IUPACData.py`` - or at very least use those values in the unit tests.

This would also fix the rounding issues which would occur by defining ``1/3`` and ``2/3`` to only two decimal plates, which is not enough.

Reply:
I've modified it so that the values are calculated based on IUPAC tables on module import